sample_data_config/web-api/web-api-data-pipeline.config.yaml

```diff
@@ -20,21 +20,71 @@ webApi:
         - items
   #elifescience profile api
   - dataset: '{ENV}'
-    table: atest_profile_api
+    table: test_people_api
     dataUrl:
-      urlExcludingConfigurableParameters: https://api.elifesciences.org/profiles?
+      urlExcludingConfigurableParameters: https://api.elifesciences.org/people?
       configurableParameters:
         pageSizeParameterName: per-page
         defaultPageSize: 100
         pageParameterName: page
     response:
       itemsKeyFromResponseRoot:
         - items
+  # civi export
+  - dataset: '{ENV}'
+    table: civi_data_export
+    stateFile:
+      bucketName: '{ENV}-elife-data-pipeline'
+      objectName: 'airflow-config/generic-web-api/{ENV}-civi-export-processing-state.json'
+    urlSourceType:
+      name: 'civi'
+      sourceTypeSpecificValues:
+        fieldsToReturn:
```

Comment:
Is your suggestion to limit the fields to our use-case and extend later rather than try to just import everything, which may cause issues with GDPR?

Reply:
we can import only the fields we need right now like you suggested and extend it later on. To be GDPR compliant if we import some PII in the  civi export table, we can run another pipeline which cleans up the table periodically.

Reply:
Okay, just good to also understand the motivation behind configuring the list of fields.

Would be good to get more thoughts from you on the cleanup you mentioned.
Would we the reset the start modified date state, re-import everything and then delete anything we imported before starting that. Although now that I typed that, I guess we would just import the `contact_is_deleted` flag and delete anything at our end where this is set?

Reply:
run a pipeline that deletes row with id's  whose latest record's `contact_is_deleted` is set to true